Google.Ads.GoogleAds/examples/AdvancedOperations/AddAdCustomizer.cs

```diff
@@ -126,337 +113,190 @@ public void Run(GoogleAdsClient client, long customerId, long[] adGroupIds)
-                Console.WriteLine($"Added feed item target with resource name " +
-                    $"'{response.Results[0].ResourceName}'.");
-            }
-        }
-        // [END add_ad_customizer_5]
-
-        /// <summary>
-        /// Creates expanded text ads that use the ad customizer feed to populate the placeholders.
-        /// </summary>
-        /// <param name="client">The Google Ads client.</param>
-        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
-        /// <param name="adGroupIds">The ad group IDs in which to create the ads.</param>
-        /// <param name="feedName">Name of the feed.</param>
-        // [START add_ad_customizer_6]
         private void CreateAdsWithCustomizations(GoogleAdsClient client, long customerId,
-            long[] adGroupIds, string feedName)
+            long adGroupId, string stringCustomizerName, string priceCustomizerName)
         {
             // Get the AdGroupAdServiceClient.
             AdGroupAdServiceClient adGroupAdService =
                 client.GetService(Services.V11.AdGroupAdService);
 
-            // Creates an expanded text ad using the feed attribute names as placeholders.
+            // Creates a responsive search ad using the feed attribute names as placeholders.
             Ad ad = new Ad()
             {
-                ExpandedTextAd = new ExpandedTextAdInfo()
+                ResponsiveSearchAd = new ResponsiveSearchAdInfo()
                 {
-                    HeadlinePart1 = $"Luxury cruise to {{={feedName}.Name}}",
-                    HeadlinePart2 = $"Only {{={feedName}.Price}}",
-                    Description = $"Offer ends in {{=countdown({feedName}.Date)}}!"
+                    Headlines =
+                    {
+                        new AdTextAsset()
+                        {
+                            Text = $"Luxury cruise to {{CUSTOMIZER.{stringCustomizerName}:Venus}}",
+                            PinnedField = ServedAssetFieldTypeEnum.Types.ServedAssetFieldType.Headline1
+                        },
+                        new AdTextAsset() { Text = $"Only {{CUSTOMIZER.{priceCustomizerName}:10.0€}}" },
+                        new AdTextAsset()
+                        {
+                            Text = $"Cruise to {{CUSTOMIZER.{stringCustomizerName}:Venus}} for {{CUSTOMIZER.{priceCustomizerName}:10.0€}}"
+                        }
+                    },
+                    Descriptions =
+                    {
+                        new AdTextAsset() { Text = $"Tickets are only {{CUSTOMIZER.{priceCustomizerName}:10.0€}}!" },
+                        new AdTextAsset() { Text = $"Buy your tickets to {{CUSTOMIZER.{stringCustomizerName}:Venus}} now!" }
+                    }
                 },
                 FinalUrls = { "http://www.example.com" }
             };
 
             List<AdGroupAdOperation> adGroupAdOperations = new List<AdGroupAdOperation>();
```

Comment:
Since you're only creating one AdGroupAd now, WDYT of creating this collection in-line in the `mutate` call?

Reply:
Done.